ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -749,9 +750,9 @@ public boolean charge() {
       try {
         quotaChargeCallback.charge(chunkSize);
         isCharged = true;
-      } catch (RouterException rEx) {
+      } catch (QuotaException quotaException) {
         logger.warn(String.format("Quota charging failed in GetBlobOperation for blob %s due to %s ", blobId.toString(),
-            rEx.toString()));
+            quotaException.toString()));
```

Comment:
nit: you don't have to use s String.format method here, logger has its own formatting.